capability-providers/keyvalue/rust/src/main.rs

```diff
@@ -112,7 +114,9 @@ fn main() -> Result<()> {
     let _ = env_logger::try_init();
 
     // Host data is a JSON object that is base64 encoded
-    let host_data_enc = std::env::var("WASMCLOUD_HOST_DATA")?;
+    let stdin = io::stdin();
+    let host_data_enc = stdin.lock().lines().next().unwrap().unwrap();
```

Comment:
I know this is still prototype provider code, but I would feel better without the double unwrap if it was something like:
```
let stdin = io::stdin();
let mut handle = stdin.lock();
let host_data_enc = String::new();
handle.read_to_string(&mut host_data_enc)?;
```

Just so that the error bubbles up instead of panicking, and since we are only reading one line. Thoughts?

Reply:
(I ripped that example from https://doc.rust-lang.org/std/io/struct.Stdin.html#examples-1). According to https://doc.rust-lang.org/std/io/trait.BufRead.html#method.read_line it reads until the newline byte